docs/userguide/ui-in-code.md

```diff
@@ -0,0 +1,362 @@
+
+        private func update(with screen: ContainerScreen) {
+            leadingContentViewController.update(
+                screen: screen.leadingScreen,
+                environment: environment
+            )
+            trailingContentViewController.update(
+                screen: screen.trailingScreen,
+                environment: environment
+            )
+
+            // Intentional force of layout pass after updating the child view controllers
+            view.layoutIfNeeded()
+        }
+
+        override internal func viewDidLoad() {
+            /** Lay out the two children horizontally, nothing workflow specific here. */
+
+            update(with: screen)
+        }
+
+        override internal func viewDidLayoutSubviews() {
+            /** Calculate the layout, nothing workflow specific here. */
+        }
+    ```
+
+    The interesting thing here is the use of `DescribedViewController` to display the nested `leadingContent` and `trailingContent` Screens.
+    `DescribedViewController` uses `Screen.viewControllerDescription` to build a new `UIViewController` if it needs to, or update an existing one if it can.
+    Everything else is just run of the mill iOS view code.
+
+=== "Android Classic"
+
+    ```kotlin title="SplitScreen.kt"
+    data class SplitScreen<L: Screen, T: Screen>(
+      val leadingScreen: L,
+      val trailingScreen: T
+    ): AndroidScreen<SplitScreen<L, T>> {
+      override val viewFactory: ScreenViewFactory<SplitScreen<L, T>> =
+        fromViewBinding(SplitScreenBinding::inflate) { screen, _ ->
+          leadingStub.show(leadingScreen)
+          trailingStub.show(trailingScreen)
+        }
+    }
+    ```
+    ```xml title="split_screen.xml"
+    <?xml version="1.0" encoding="utf-8"?>
+    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="horizontal"
+        >
+
+      <com.squareup.workflow1.ui.WorkflowViewStub
+          android:id="@+id/leading_stub"
+          android:layout_width="0dp"
+          android:layout_height="match_parent"
+          android:layout_weight="30"
+          />
+
+      <View
+          android:layout_width="1dp"
+          android:layout_height="match_parent"
+          android:background="@android:drawable/divider_horizontal_bright"
+          />
+
+      <com.squareup.workflow1.ui.WorkflowViewStub
+          android:id="@+id/trailing_stub"
+          android:layout_width="0dp"
+          android:layout_height="match_parent"
+          android:layout_weight="70"
+          />
+
+    </LinearLayout>
+    ```
+
+=== "Android Jetpack Compose"
+    ```kotlin title="SplitScreen.kt"
+    data class SplitScreen<L: Screen, T: Screen>(
+      val leadingScreen: L,
+      val trailingScreen: T
+    ): ComposeScreen<SplitScreen<L, T>> {
+      @Composable override fun Content(viewEnvironment: ViewEnvironment) {
+        TODO()
```

Comment:
Any generous soul have a moment to fill in the blank here?

Reply:
```suggestion
Row {
  WorkflowRendering(
    rendering = leadingScreen,
    modifier = Modifier
      .weight(1 / 3f)
      .fillMaxHeight()
  )
  WorkflowRendering(
    rendering = trailingScreen,
    modifier = Modifier
      .weight(2 / 3f)
      .fillMaxHeight()
  )
}
```